### PR TITLE
v1.25 api reference multi pages

### PR DESCRIPTION
**Pull request by feloy**
Update of API Reference multi-pages (https://kubernetes.io/docs/reference/kubernetes-api/) for v1.25

### CODE REVIEW

File: content/en/docs/reference/kubernetes-api/common-definitions/quantity.md [hunk 1]
```diff
@@ -30,27 +30,33 @@ Quantity is a fixed-point representation of a number. It provides convenient mar
...
 No matter which of the three exponent forms is used, no quantity may represent a number greater than 2^63-1 in magnitude, nor may it have more than 3 decimal places. Numbers larger or more precise will be capped or rounded up. (E.g.: 0.1m will rounded up to 1m.) This may be extended in the future if we require larger or smaller quantities.
 
 When a Quantity is parsed from a string, it will remember the type of suffix it had, and will use the same type again when it is serialized.
 
 Before serializing, Quantity will be put in "canonical form". This means that Exponent/suffix will be adjusted up or down (with a corresponding increase or decrease in Mantissa) such that:
-  a. No precision is lost
-  b. No fractional digits will be emitted
-  c. The exponent (or suffix) is as large as possible.
+
+- No precision is lost - No fractional digits will be emitted - The exponent (or suffix) is as large as possible.
```
Comment (sftim) [inline content/en/docs/reference/kubernetes-api/common-definitions/quantity.md:53]: BTW we're turning a list into not-a-list here.

File: content/en/docs/reference/kubernetes-api/common-definitions/quantity.md [hunk 2]
```diff
@@ -30,27 +30,33 @@ Quantity is a fixed-point representation of a number. It provides convenient mar
...
+
 The sign will be omitted unless the number is negative.
 
 Examples:
-  1.5 will be serialized as "1500m"
-  1.5Gi will be serialized as "1536Mi"
+
+- 1.5 will be serialized as "1500m" - 1.5Gi will be serialized as "1536Mi"
```
Comment (sftim) [inline content/en/docs/reference/kubernetes-api/common-definitions/quantity.md:59]: BTW we're turning a list into not-a-list here.

**Comment by sftim** [comment]
Actually, the deadline for _these_ docs is basically release day.

File: content/en/docs/reference/kubernetes-api/common-definitions/quantity.md (hunk 1 above)
Comment (feloy) [reply content/en/docs/reference/kubernetes-api/common-definitions/quantity.md:53]: This is due to the changes in the doc of the original source code. I'm not sure we can do anything here.

We can also see the side effect of the changes done for the Quantity definition, adding triple quotes around the definition.

The problem is that the doc lines are all concatenated into a single line to create the definition in the swagger file, adding \n's when blank lines are found, but no other markdown are respected.

Comment (feloy) [reply content/en/docs/reference/kubernetes-api/common-definitions/quantity.md:53]: related commit on source code: https://github.com/kubernetes/kubernetes/commit/971046e26e830f9b53219c7b0a078213b338572c

Comment (sftim) [reply content/en/docs/reference/kubernetes-api/common-definitions/quantity.md:53]: OK, that's a clear explanation.

**Comment by reylejano** [comment]
Thank you for generating the API reference docs 
/approve

**Review by kcmartin — APPROVED** [review]
Thank you!

/lgtm
/approve

**Comment by ttloves2** [comment]
A better world for all

**Pull request merged**